README.md

```diff
@@ -127,8 +120,8 @@ If you find a bug, please report big and small bugs by [opening an issue](https:
 
 <p align="center">
 	<a href="https://eoscostarica.io/">
-		<img src="https://raw.githubusercontent.com/eoscostarica/design-assets/master/logos/eosCR/fullColor-horizontal-transparent-white.png"
-		width="400px" >
+		<img src="https://user-images.githubusercontent.com/5632966/178800859-e05a07ff-b3e5-41a9-9137-d1a271b958ff.png"
```

Comment:
please update image URL to this one: https://raw.githubusercontent.com/eoscostarica/.github/master/.github/workflows/images/eos-costa-rica-logo.png

Reply:
Done.